### DISCUSSION TITLE
Rocketmq Dashboard 查看消息详情异常。

### DISCUSSION THREAD

**Discussion by AI1186780944** [apache/rocketmq#7108] [Q&A]
服务版本： rocketmq：5.1.3。
Dashboard版本：1.0.0。
服务运行模式：Local模式，开启proxy。
场景描述：
Rocketmq Dashboard的namesrvAddr指向是rocketmq-namesrv地址（无法指向proxy的remoting或者grpc端口，会发生错误）。
问题：
当客户端生产者在使用proxy代理的情况下，发送的消息。想通过Rocketmq Dashboard查看消息的消费情况，会出现”ERROR Occur service exception: Failed to query message by Id: 0108719066FF9A489C04DB99CA00000003“的异常错误。但是如果客户端生产者不指向proxy，而是直接向namesrv发送的消息，这种情况下是可以正常查看消息的消费情况的。为什么会出现这种现象？

**Comment by onejimmyboy** [comment]
rocketmq-dashboard1.0.0版本依赖的rocketmq-tools的版本是4.9.0的，没有引入最新的5.1.3。rocketmq-dashboard年都没怎么更新了
